updater/src/main.rs

```diff
@@ -360,14 +361,15 @@ impl Cities {
             cities
         } else {
             let mut f = File::create(&path).unwrap();
-            writeln!(f, "runtime: seattle,berlin,krakow").unwrap();
```

Comment:
This `else` branch is creating the default updater config for people who haven't opted into certain cities yet. I don't think we should add `leeds` here, because it'll increase the default release's size more. I'm pretty close to a more user-friendly solution for letting people opt into "extra optional" maps through the game UI. But until then, locally you would edit `data/config` to include leeds, but not check this change in.

https://dabreegster.github.io/abstreet/dev/index.html#downloading-more-cities has the instructions for how users can opt into extra content

Reply:
:+1: